cli/state.go

```diff
@@ -1086,18 +1088,18 @@ func printInternalExecutionsHtml(trace []*types.ExecutionResult, getCode func(ad
 		slow := im.Duration > 10*time.Millisecond
 		veryslow := im.Duration > 50*time.Millisecond
 
-		fmt.Printf(`<div class="exec">
-<div><h4 class="call">%s:%s</h4></div>
+		fmt.Printf(`<div class="exec" id="%s">
+<div><a href="#%s"><h4 class="call">%s:%s</h4></a></div>
 <div><b>%s</b> -&gt; <b>%s</b> (%s FIL), M%d</div>
 %s
 <div><span class="slow-%t-%t">Took %s</span>, <span class="exit%d">Exit: <b>%d</b></span>%s
-`, codeStr(toCode), methods[toCode][im.Msg.Method].name, im.Msg.From, im.Msg.To, types.FIL(im.Msg.Value), im.Msg.Method, params, slow, veryslow, im.Duration, im.MsgRct.ExitCode, im.MsgRct.ExitCode, ret)
+`, hashName, hashName, codeStr(toCode), methods[toCode][im.Msg.Method].name, im.Msg.From, im.Msg.To, types.FIL(im.Msg.Value), im.Msg.Method, params, slow, veryslow, im.Duration, im.MsgRct.ExitCode, im.MsgRct.ExitCode, ret)
```

Comment:
Is there a reason why `hashName` is repeated here?

Reply:
It's used 2 times, for the anchor in div, and the link